metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -99,7 +105,11 @@ class PlatformDetail:
     )
 
 
-class PowerBiAPIConfig(EnvBasedSourceConfigBase):
+class PowerBiDashboardSourceConfig(StatefulIngestionConfigBase):
+    platform_name: str = "powerbi"
+
+    platform_urn: str = builder.make_data_platform_urn(platform=platform_name)
```

Comment:
Do we need to expose ` platform_name` and  `platform_urn` as user configs ? If not, we should keep these hidden or move it somewhere else. You can use `hidden_from_schema` like [this](https://github.com/datahub-project/datahub/blob/4d8e1f5e782956b778880b7864ccee7240532cab/metadata-ingestion/src/datahub/ingestion/source/sql/hive.py#L95)